admin/class-yoast-form.php

```diff
@@ -445,17 +447,25 @@ public function select( $var, $label, array $select_options ) {
 			)
 		);
 
-		$select_name   = esc_attr( $this->option_name ) . '[' . esc_attr( $var ) . ']';
-		$active_option = ( isset( $this->options[ $var ] ) ) ? $this->options[ $var ] : '';
+		$select_name       = esc_attr( $this->option_name ) . '[' . esc_attr( $var ) . ']';
+		$active_option     = ( isset( $this->options[ $var ] ) ) ? $this->options[ $var ] : '';
+		$wrapper_start_tag = '';
+		$wrapper_start_end = '';
```

Comment:
`wrapper_start_end` is sending me mixed signals about what it is :). I'd suggest $wrapper_start_tag and $wrapper_end_tag.